homeassistant/components/modbus/climate.py

```diff
@@ -87,6 +89,33 @@ def __init__(
         self._attr_target_temperature_step = config[CONF_TARGET_TEMP]
```

Comment:
Though this part remains the same, there is a question.
line 89 and line 90 have the same lvalue. Maybe line 89 should be self._attr_target_temperature = config[CONF_TARGET_TEMP]?